freqtrade/rpc/local_rest_server.py

```
@@ -0,0 +1,113 @@
+            #         /status /table /performance /count,
+            #         /start /stop /help'
+
+            rest_cmds ='Commands implemented: <br>' \
+                       '<a href=/daily?timescale=7>/daily?timescale=7</a>' \
+                       '<br>' \
+                       '<a href=/stop>/stop</a>' \
+                       '<br>' \
+                       '<a href=/start>/start</a>'
+            return rest_cmds
+
+        @app.route('/daily', methods=['GET'])
+        def daily():
+            try:
+                timescale = request.args.get('timescale')
+                logger.info("LocalRPC - Daily Command Called")
+                timescale = int(timescale)
+
+                stats = self._rpc_daily_profit(timescale,
+                                               self._config['stake_currency'],
+                                               self._config['fiat_display_currency']
+                                               )
+
+                stats = dumps(stats, indent=4, sort_keys=True, default=str)
+                return stats
+            except RPCException as e:
+                return e
+
+        @app.route('/start', methods=['GET'])
+        def start():
+            """
+            Handler for /start.
+            Starts TradeThread
+            """
+            msg = self._rpc_start()
+            print("msg is", msg)
+            return msg
+
+        @app.route('/stop', methods=['GET'])
+        def stop():
+            """
+            Handler for /stop.
+            Stops TradeThread
+            """
+            msg = self._rpc_stop()
+            print("msg is", msg)
+            return msg
+
+        """
+        Section to handle configuration and running of the Rest serve
+        also to check and warn if not bound to 127.0.0.1 as a security risk.
+        """
+
+        rest_ip = self._config['rest_cmd_line']['listen_ip_address']
+        rest_port = self._config['rest_cmd_line']['listen_port']
```

Please add new config values to the jsonschema in `freqtrade/constants.py CONF_SCHEMA` for proper input validation. It should be possible to check for `hostname`, `ipv4` and `ipv6` formats out of the box with jsonschema, but I'm not sure on that.

---

This test was failing silently when added a bad IP address. 

ipv4,6 hostname need format-checker loaded to do this correctly.
I've modified configuration.py to this end and tested ipv4  good and bad, which acts as would hope.